server/rpm/pbench-server.spec.j2

```diff
@@ -125,14 +125,14 @@ fi
 /%{installdir}/bin/pbench-base.sh
 
 %defattr(755, pbench, pbench, 755)
-/%{installdir}/bin/getconf.py
 /%{installdir}/bin/pbench-audit-server
 /%{installdir}/bin/pbench-audit-server.sh
 /%{installdir}/bin/pbench-base.py
 /%{installdir}/bin/pbench-check-tb-age
 /%{installdir}/bin/pbench-check-tb-age.py
 /%{installdir}/bin/pbench-clean-up-dangling-results-links
 /%{installdir}/bin/pbench-clean-up-dangling-results-links.sh
+/%{installdir}/bin/pbench-server-config
```

Comment:
Why did you insert this at line 135 instead of, say, at line 141?

Reply:
I'll fix in another commit shortly.